src/aig-common/old-common/services/cpv.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Resolve } from '@angular/router';
```

Comment:
the file position not is correct. take ispiration from the position of this: [https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/generic/resolver/eopoo-type.resolver.ts](url)